pkgs/applications/networking/bee/bee-clef.nix

```diff
@@ -0,0 +1,71 @@
+{ version ? "release", stdenv, lib, substituteAll
```

Comment:
What is the motivation for making this configurable as only one version is supported?

Reply:
ok, deleted it